Automattic/simplenote-ios
# Search Highlighting out of place after rotation

### Fix
This PR fixes #1255 

When you load a note while search is enabled all of the instances of the search term are highlighted in that note and the current instance is given a special highlight to make it stand out.  Currently if you rotate the device while looking at the note editor in search mode the special highlighting becomes out of place.

In this PR, when the device is rotated, the current highlight is removed and then replaced with a new one in the correct location.

### Test
1. Search for some text that is in one of your notes, open the note and notice that the first instance of the search term is highlighted
2. rotate the device
3. notice that the highlight is still correctly highlighted after rotation

### Review
***(Required)*** Add instructions for reviewers.  For example:
> Only one developer is required to review these changes, but anyone can perform the review.

### Release
> `RELEASE-NOTES.txt` was updated in fe34044e with:
> 
> > -   Some bug fixes #1345, #1347, #1348
